chainercv/evaluations/eval_semantic_segmentation.py

```diff
@@ -59,15 +59,13 @@ def eval_semantic_segmentation(pred_label, gt_label, n_class):
     Args:
         pred_label (array): An integer array of image containing
             class labels as values, which is obtained from inference.
-            This has shape :math:`(N, 1, H, W)` or :math:`(1, H, W)`,
+            This has shape :math:`(N, H, W)` or :math:`(H, W)`,
             where :math:`N` is size of the batch, :math:`H` is the height
             and :math:`W` is the width.
```

Comment:
`H` is the height and W` is the width` -> `H and W are the height and width of the image`